src/mmwtex.c

```diff
@@ -1261,12 +1261,12 @@ void printTexHeader(flag texHeaderFlag)
     if (texHeaderFlag && !g_oldTexFlag) {
       /* LaTeX 2e */
       print2("\\documentclass{article}\n");
+      print2("\\usepackage{amssymb} %% amssymb must be loaded after phonetic\n");
       print2("\\usepackage{phonetic} %% for \\riota\n");
```

Comment:
here and below, you probably mean "before"

Reply:
This was already fixed in commit [bd1473a](https://github.com/metamath/metamath-exe/pull/133/commits/bd1473ada6615dd54a43fd9a11c1f33bbdf6c632) "fixed comment mistake"